src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -55,7 +55,7 @@ export const WebhookReceiverPost = async (request: Request, response: Response):
 				})
 			}
 		});
-		webhookRouter(webhookContext);
+		await webhookRouter(webhookContext);
```

Comment:
I think, we should not wait for message to process successfully  before sending response back to webhook originator.

Reply:
yes, we should.  This is how webhooks should work.  If you don't process it correctly, it should fail.  Webhooks sometimes have retry mechanism if they don't work and it comes down to the implementer to do the right thing - either process it immediately or add it to queue to be processed later both of which needs to fail outright if it doesn't work as we can then audit the trail.